buildbot.mariadb.org/scripts/docker-library-build-and-test.sh

```diff
@@ -185,7 +181,7 @@ buildmanifest() {
 
 devmanifest=mariadb-devel-${container_tag}-$commit
 
-buildmanifest mariadb-devel $devmanifest $container
+buildmanifest mariadb-devel "$devmanifest" "$container"
 
```

Comment:
its about this point we need a `buildah rm "${container}"` in the `EXIT` trap to remove the in build container.